proto/attestation_policy/v1alpha1/attestation_policy.proto

```diff
@@ -31,6 +31,8 @@ message APKubernetes {
   // Valid template components:
   // {{ .ClusterName }} - Name of cluster
   // {{ .PodMeta.Namespace }} - Namespace of the pod
+  // {{ index .PodMeta.Labels "key" }} - Pod label value of a provided key
+  // {{ index .PodMeta.Annotations "key" }} - Pod annotation value of a provided key
```

Comment:
Do we need to mention that this is supported in Connect only? And also that OSS cofidectl supports the full clusterspiffeid format?